roles/download/defaults/main.yml

```diff
@@ -447,9 +447,12 @@ haproxy_image_tag: 2.3
 
 # Coredns version should be supported by corefile-migration (or at least work with)
 # bundle with kubeadm; if not 'basic' upgrade can sometimes fail
-coredns_version: "1.7.0"
-coredns_image_repo: "{{ kube_image_repo }}/coredns"
-coredns_image_tag: "{{ coredns_version }}"
+coredns_current_kube_version: "{{ kube_version | regex_replace('^v(?P<major>\\d+).(?P<minor>\\d+).(?P<patch>\\d+)$', '\\g<major>.\\g<minor>') }}"
```

Comment:
If think you can use https://github.com/kubernetes-sigs/kubespray/blob/master/roles/download/defaults/main.yml#L90

Reply:
you're right